Bobcat/Profile.cpp

```diff
@@ -350,6 +352,42 @@ void Profiles::Add()
 	Sync();
 }
 
+void Profiles::Clone()
+{
+	if(String s; EditTextNotNull(s, t_("Clone Profile"), t_("Name"))) {
+		if(!list.FindSetCursor(s)) {
+			Profile source = list.Get(1).To<Profile>();
+			Profile p(source);
+			p.name = s;
+			auto& m = GetHyperlinkPatterns();
+			int i = m.FindAdd(p.name);
+			m[i] <<= m.Get(source.name);

```

Comment:
I wonder why are the Linkifier patterns stored outside of the Profile class? It seems to complicate the code quite a lot (not just here, also in serialization).

Reply:
Yeah, lazy coding again. Profile management was not my priority. 
The alternative might be to load the specific profile and fetch the list only when a search is issued. It might be slightly slower approach, but surely it will consume less memory and make the profile management easier.